resolwe/flow/management/commands/list_docker_images.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+from resolwe.flow.models import Process
+
+
+class Command(BaseCommand):
+	help = "List Docker images used by processes in a format suitable for use in Ansible"
+
+	def handle(self, *args, **options):
+		# This Docker image is used if a process doesn't specify its own
+		default_docker_image = settings.FLOW_EXECUTOR['CONTAINER_IMAGE']
+
+		# Gather a list of all custom Docker images the processes are using
+		unique_docker_images = set(p.requirements['executor']['docker']['image'] for p in Process.objects.filter(requirements__icontains='docker'))
+
+		unique_docker_images.add(default_docker_image)
+
+		# Assemble a list of Docker images suitable for passing on to Ansible
+		out = ''
+
+		for i in unique_docker_images:
+			# Split Docker image specifier into image name and tag
+			s = i.split(':')
+			img = s[0]
+			tag = s[1] if len(s) == 2 else 'latest'
+
+			if out:
+				out += ', '
+
+			out += "{ 'name': '" + img + "', 'tag': '" + tag + "' }"
```

Comment:
I would add an argument to select the output format (`--format`), with default to `plain` (output image names with tag, one per line) and `yaml`.